docker/build_deps.sh

```diff
@@ -37,17 +45,21 @@ configure() {
 
 build_bde() {
     pushd srcs/bde
-    bbs_build configure
-    bbs_build build -j8
+    if [ "$install_only" = false ]; then
+        bbs_build configure
+        bbs_build build -j8
+    fi
     bbs_build --install=/opt/bb --prefix=/ install
     popd
 }
 
 build_ntf() {
```

Comment:
This `build_ntf` function causes ntf to rebuild from scratch every time this script is called, no matter if incremental build object exist. This rebuild takes 5 minutes.

`build_bde`, on the other hand, doesn't recompile everything.

If we can get ntf not to rebuild everything, we can even omit this `install_only` flag.

Reply:
I'm looking into this now, I will add some comments with what I find.

Reply:
By default, ntf's `configure` script cleans build artifacts before it does anything.  When it's passed the `--keep` flag it refrains from doing this.